src/platform/forms-system/src/js/containers/FormApp.jsx

```diff
@@ -3,16 +3,28 @@ import { connect } from 'react-redux';
 import Scroll from 'react-scroll';
 import PropTypes from 'prop-types';
 import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-
 import { isLoggedIn } from 'platform/user/selectors';
 
+import BackLink from '../components/BackLink';
 import FormNav from '../components/FormNav';
 import FormTitle from '../components/FormTitle';
 import { isInProgress, hideFormTitle } from '../helpers';
 import { setGlobalScroll } from '../utilities/ui';
 
 const { Element } = Scroll;
 
+const BeforePage = ({ useTopBackLink }) => {
+  return useTopBackLink ? <BackLink /> : null;
+};
+
+const BeforePageContent = connect(state => ({
+  useTopBackLink: state?.form?.layout?.useTopBackLink,
+}))(BeforePage);
```

Comment:
I've never seen `connect` used within a component. We are using a new enough version of redux that includes [`useSelector`](https://react-redux.js.org/api/hooks#useselector). If we use a form global `useTopBackLink`, FormApp does have access to the `formConfig`. If you want to keep it per page then I think we can use a combo of `currentLocation` and `pageList` to find the page config.

Reply:
Great callout - relying on formConfig seems like a good alternate, and can scrap redux altogether.